pkg/datapath/linux/probes/probes.go

```diff
@@ -196,10 +203,25 @@ func (p *ProbeManager) SystemKernelHz() (int, error) {
 // warnings when optional parameters are not enabled.
 func (p *ProbeManager) SystemConfigProbes() error {
 	config := p.features.SystemConfig
-	// Required
-	if !config.ConfigBpf.Enabled() {
-		return fmt.Errorf("CONFIG_BPF kernel parameter is required")
+
+	// Check Kernel Config is available or not.
+	// We are replicating BPFTools logic here to check if kernel config is available
+	// https://elixir.bootlin.com/linux/v5.7/source/tools/bpf/bpftool/feature.c#L390
+	info := unix.Utsname{}
+	err := unix.Uname(&info)
+	var release string
+	if err == nil {
+		release = strings.TrimSpace(string(bytes.Trim(info.Release[:], "\x00")))
+	}
+
+	// Any error checking these files will return Kernel config not found error
+	if _, err := os.Stat(fmt.Sprintf("/boot/config-%s", release)); err != nil {
```

Comment:
If `unix.Uname` returns an err at the line 211, we will do stat on `/boot/config-`. Is it OK?

Reply:
Should we check for the remaining parameters if `unix.Uname` errors or should we just return an error?

Reply:
@brb ^

Reply:
Probably just return an error. Did you check what bpftool does in this situation?

Reply:
It treats it as the same error as the config is not available. I can return the same error here.